test/index.js

```diff
@@ -228,21 +139,14 @@ describe('Bloom', function() {
     });
 
     it('use the max size', function() {
```

Comment:
@semaj it doesn't appear like this test is relevant anymore now that there is no max size?